src/pages/public_registration/action_accordions/DeletePublication.tsx

```diff
@@ -0,0 +1,160 @@
+import { Registration, UnpublishPublicationRequest } from '../../../types/registration.types';
+import { Box, Breadcrumbs, Button, DialogActions, Divider, TextField, Typography } from '@mui/material';
+import { useState } from 'react';
+import { Modal } from '../../../components/Modal';
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+import { useTranslation } from 'react-i18next';
+import { FindRegistration } from './FindRegistration';
+import { unpublishRegistration } from '../../../api/registrationApi';
+import { setNotification } from '../../../redux/notificationSlice';
+import { useDispatch } from 'react-redux';
+import { LoadingButton } from '@mui/lab';
+import { isErrorStatus, isSuccessStatus } from '../../../utils/constants';
+import { RequiredDescription } from '../../../components/RequiredDescription';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { useHistory } from 'react-router-dom';
+
+interface DeleteForm {
+  deleteMessage: string;
+}
+
+interface DeletePublicationProps {
+  registration: Registration;
+}
+
+export const DeletePublication = ({ registration }: DeletePublicationProps) => {
+  const [showDeleteModal, setShowDeleteModal] = useState(false);
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const [selectedDuplicate, setSelectedDuplicate] = useState<Registration | null>(null);
+  const [awatingUnpublishedResponse, setAwaitingUnpublishingResponse] = useState(false);
+  const history = useHistory();
+
+  const deleteValidationSchema = Yup.object().shape({
+    deleteMessage: Yup.string()
+      .min(3, t('unpublish_actions.unpublish_comment_min_length_feedback'))
+      .required(t('unpublish_actions.unpublish_comment_required')),
```

Comment:
Vi har noen generelle oversettelser for feedback, sånn at de blir ulike overalt. Ser vi bare har gjenbrukt obligatorisk-tekst når noe er for kort andre plasser faktisk, så kan godt legge inn en ny nøkkel for "{{field}} må ha minst {{count}} tegn" eventuelt

```suggestion
      .min(3, t('feedback.validation.is_required', {
    field: i18n.t('common.justification'),
  })
      .required(t('feedback.validation.is_required', {
    field: i18n.t('common.justification'),
  }),
```